client/src/Pages/Registratie.tsx

```diff
@@ -150,33 +197,49 @@ class Registratie extends React.Component<IProps,IState>{
         });
 
         // Voeg de gebruiker toe aan de database.
-        console.log("sending");
-        await fetch(this.props.apiLink+"/addgebruiker",{
+        let addgebruiker: any = await fetch(this.props.apiLink+"/addgebruiker",{
             method:'POST',
             body:formData
-        }).then(value => console.log(value));
+        }).then(res => res.json());
+        // Verwerk de feedback vanuit de server.
+        this.setState({addgebruikerSuccess: addgebruiker.addgebruikerSuccess});
 
         // Maak een nieuw rooster aan in de database.
         if (this.state.isWerkgever) {
-            fetch(this.props.apiLink + "/addrooster", {
+            let addrooster: any = await fetch(this.props.apiLink + "/addrooster", {
                 headers: {'Content-Type': 'application/json'},
                 method: 'POST',
                 body: JSON.stringify({roosterName: this.state.roosterName, koppelCodeWerkgever: this.state.koppelCodeWerkgever, email: this.state.email})
-            });
+            }).then(res => res.json());
+            // Verwerk de feedback vanuit de server.
+            this.setState({addroosterSuccess: addrooster.addroosterSuccess});
         }
+
+        // Koppel een gebruiker aan het rooster dat bij zijn koppelcode hoort.
         if (!this.state.isWerkgever) {
-            fetch(this.props.apiLink + "/koppelgebruiker", {
+            let koppelgebruiker: any = await fetch(this.props.apiLink + "/koppelgebruiker", {
                 headers: {'Content-Type': 'application/json'},
                 method: 'PUT',
                 body: JSON.stringify({email: this.state.email, koppelCodeWerknemer: this.state.koppelCodeWerknemer})
-            });
+            }).then(res => res.json());
+            // Verwerk de feedback vanuit de server.
+            this.setState({koppelgebruikerSuccess: koppelgebruiker.koppelgebruikerSuccess})
+        }
+
+        // Geef door dat de registratie succesvol is wanneer...
+        if (this.state.isWerkgever && this.state.addgebruikerSuccess && this.state.addroosterSuccess) {
+            this.setState({registratieSucces: true});
+            alert("Registratie succesvol.");
+        } else if (!this.state.isWerkgever && this.state.addgebruikerSuccess && this.state.koppelgebruikerSuccess) {
+            this.setState({registratieSucces: true});
+            alert("Registratie succesvol.");
```

Comment:
Als de registratie niet succesvol is gebeurt er niks (dit mag je wel zo laten  want je checkt al goed of alles wel goed naar de db gaat, maar is alleen een idee)